languages/ruby/exercises/concept/exceptions/.meta/example.rb

```diff
@@ -0,0 +1,16 @@
+class SimpleCalculator
+  ALLOWED_OPERATIONS = ['+', '-', '/', '*']
+
+  UnsupportedOperation = Class.new(StandardError)
+
+  def self.calculate(first_operand, second_operand, operation)
+    raise ArgumentError.new unless (first_operand.is_a?(Integer) && second_operand.is_a?(Integer))
+    raise UnsupportedOperation.new unless ALLOWED_OPERATIONS.include?(operation)
+
+    result = first_operand.to_i.public_send(operation, second_operand.to_i)
```

Comment:
I don't think I would use `to_i` here as if I gave any other things, it would cause surprises.  And when teaching exceptions, we don't want surprising behavior on top of the exceptions.

Also, since the guard happens on line 7, it would be surprising if a "not integer" ever got to this line.  So the changes should probably be:

```ruby
first_operand.public_send(operation, second_operand)
```

Reply:
👍 the `to_i` is a left over from a previous version